pleasant/helpers.go

```diff
@@ -434,7 +434,9 @@ func TrimDoubleQuotes(str string) string {
 }
 
 func Unescape(str string) string {
-	return strings.ReplaceAll(str, `\\`, `\`)
+	str = strings.ReplaceAll(str, `\"`, `"`)
+	str = strings.ReplaceAll(str, `\\`, `\`)
+	return str
```

Comment:
@ramiuslr thanks for adding this!

I would have a minor suggestion on how to implement this without having to execute strings.ReplaceAll multiple times:

```suggestion
	r := strings.NewReplacer(`\\`, `\`, `\"`, `"`)
	return r.Replace(str)
```

This would perform both replacements in one go (and allow for adding new ones in the future, if required).

Reply:
Awesome, that was exactly what I was looking for, I didn't see this `NewReplacer` function !